www/caddy/src/opnsense/service/templates/Pischem/Caddy/rc.conf.d/caddy

```diff
@@ -0,0 +1,16 @@
+# DO NOT EDIT THIS FILE -- OPNsense auto-generated file
+{% if helpers.exists('Pischem.caddy.general.enabled') %}
+    {%- set general_enabled = helpers.toList('Pischem.caddy.general.enabled') | first %}
+    {%- if general_enabled == '1' %}
+caddy_enable="YES"
+# Path to the Caddy setup script
+caddy_setup="/usr/local/opnsense/scripts/Pischem/Caddy/setup.sh"
+# Execute the setup script before Caddy starts
+start_precmd="sh ${caddy_setup}"
+    {%- else %}
+caddy_enable="NO"
```

Comment:
looks a bit odd to have two times "NO" in this

Reply:
The second "NO" is a fallback condition if no Configuration is present that either enables or disables caddy so the template generation can never hard fail with an error.

Reply:
it's a bit unusual, but I don't mind if you want to keep it